### PR TITLE
PR : Merge sort algorithm in C programming

### PR DESCRIPTION
**Pull request by Vybhavigithub**
solved #77  
This is merge sorting algorithm in c language where overview ,source code and explaination have been added.

### CODE REVIEW

File: 2-algorithms/0-c-programming/MergeSort.exe
Comment (Abishethvarman) [inline 2-algorithms/0-c-programming/MergeSort.exe:1]: delete this file

File: 2-algorithms/0-c-programming/MergeSort.c
Comment (Abishethvarman) [inline 2-algorithms/0-c-programming/MergeSort.c:1]: make this in in .MD file. No need of .c file. Refer the readme files in the repo incase you need a claririty.

**Review by Abishethvarman — CHANGES_REQUESTED** [review]

File: 2-algorithms/0-c-programming/MergeSort.c
Comment (Vybhavigithub) [reply 2-algorithms/0-c-programming/MergeSort.c:1]: okay

**Review by Vybhavigithub — COMMENTED** [review]
please do review changes

**Comment by Vybhavigithub** [comment]
@Abishethvarman Are there any more changes required ? Please do review .

**Pull request closed without merging**